libraries/AP_Airspeed/AP_Airspeed.cpp

```diff
@@ -399,16 +436,137 @@ void AP_Airspeed::read(void)
     }
 #endif
 
-    // setup primary
-    if (healthy(primary_sensor.get())) {
-        primary = primary_sensor.get();
-        return;
-    }
-    for (uint8_t i=0; i<AIRSPEED_MAX_SENSORS; i++) {
-        if (healthy(i)) {
-            primary = i;
+    switch(blend_conf) {
+    case BLEND_NONE:
+        process_no_blend();
+        break;
+    case BLEND_FULL:
+        if (_ahrs == nullptr) {
```

Comment:
_ahrs should never be null, you can remove this

Reply:
I tried. Not possible to other places where AP_Airspeed exists without AHRS.